compiler/compile.bal

```diff
@@ -135,6 +141,11 @@ function subModuleSourceParts(string basename, bir:ModuleId id) returns front:So
     return sourceParts;
 }
 
+function getPathRelativeToModuleDir(string modulesDirPath, string filePath) returns string|file:Error {
```

Comment:
We want the filename in the same form as the filename for the root. So I think we just do `file:joinPath(directory, file:basename(md,absPath))`

Reply:
fixed. Add normalize to get rid of the `/./` joinPath adds